src/sentry/backup/crypto.py

```diff
@@ -135,7 +135,9 @@ def create_encrypted_export_tarball(json_export: json.JSONData, encryptor: Encry
     pem = encryptor.get_public_key_pem()
     data_encryption_key = Fernet.generate_key()
     backup_encryptor = Fernet(data_encryption_key)
-    encrypted_json_export = backup_encryptor.encrypt(json.dumps(json_export).encode("utf-8"))
+    encrypted_json_export = backup_encryptor.encrypt(
+        json.dumps_experimental("backup.enable-orjson", json_export).encode("utf-8")
```

Comment:
No need for "utf-8" since it is the default value

```suggestion
        json.dumps_experimental("backup.enable-orjson", json_export).encode()
```